CRM/Admin/Form/Setting/Path.php

```diff
@@ -36,7 +36,7 @@ class CRM_Admin_Form_Setting_Path extends CRM_Admin_Form_Setting {
     'customTemplateDir' => CRM_Core_BAO_Setting::DIRECTORY_PREFERENCES_NAME,
     'customPHPPathDir' => CRM_Core_BAO_Setting::DIRECTORY_PREFERENCES_NAME,
     'extensionsDir' => CRM_Core_BAO_Setting::DIRECTORY_PREFERENCES_NAME,
-    'ext_max_depth' => CRM_Core_BAO_Setting::EXT,
+    'ext_max_depth' => CRM_Core_BAO_Setting::DIRECTORY_PREFERENCES_NAME,
```

Comment:
According to the metadata, this is actually supposed to be 
```suggestion
    'ext_max_depth' => 'Extension Preferences',
```

See https://github.com/civicrm/civicrm-core/blob/e3723cb468c85cd7ddc2dd73b772b65bdcad228a/settings/Extension.setting.php#L41

Reply:
@ufundo can you rebase this with that change?

Reply:
done. though I think @eileenmcnaughton is working on a proper fix